scripts/globals/effects/overdrive.lua

```diff
@@ -9,18 +9,50 @@
 -----------------------------------
 
 function onEffectGain(target,effect)
-end;
+    target:addMod(MOD_OVERLOAD_THRESH, 5000)
+    local pet = target:getPet()
+    if pet then
```

Comment:
Should this not check if the pet is an automaton? Otherwise you could have the effect work on pets from /SMN or /BST.

Reply:
You're right, I just tested it. Seems Familiar and Reward from BST also allow this on the Automaton if you use pup/bst as well. I'll have to make a new lua callback to get the pet type.

Reply:
> I'll have to make a new lua callback

One already exists for this.